### ISSUE TITLE
Can't retrieve 1.2.1 from community-terraform-providers/ignition

### ISSUE THREAD

**Issue by ribbybibby** [community-terraform-providers/terraform-provider-ignition#10]
I need to use the 2.x ignition spec because [Flatcar Container Linux doesn't seem to currently support spec 3.x](https://github.com/kinvolk/ignition/tree/flatcar-master/config). However, I can't source a version of this provider that supports 2.x (`1.2.1`) from `community-terraform-providers/ignition`.

Am I fine to continue using the old provider path`terraform-providers/ignition`? I'm concerned that at some point that path may be deprecated and removed and we'll no longer be able to source `1.2.1`.

### Terraform Version
```
$ terraform -v
Terraform v0.14.2
```

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    ignition = {
      source  = "community-terraform-providers/ignition"
      version = "1.2.1"
    }
  }
}
```

### Debug Output
```
$ terraform init .

Initializing the backend...

Initializing provider plugins...
- Finding community-terraform-providers/ignition versions matching "1.2.1"...

Error: Failed to query available provider packages

Could not retrieve the list of available versions for provider
community-terraform-providers/ignition: no available releases match the given
constraints 1.2.1
```

**Comment by bgilbert** [comment]
Thanks for bringing this up.  It appears that the [original terraform-provider-ignition repo](https://github.com/hashicorp/terraform-provider-ignition) has been archived by its maintainers, and I'm [not seeing it](https://registry.terraform.io/search/providers?q=ignition) in Terraform registry search.  As a convenience to the community, we might want to publish older releases to the Terraform registry under [our fork](https://registry.terraform.io/providers/community-terraform-providers/ignition/latest).  @LorbusChris, any thoughts?

**Comment by LorbusChris** [comment]
I think that sounds feasible. I've pushed the `master` branch from the original repo into this repo as `v1`.

I'm not familiar with the release process though. I believe @yussufsh did this in the past - could you help with this?

**Comment by yussufsh** [comment]
I have created [PR](https://github.com/community-terraform-providers/terraform-provider-ignition/pull/12) for adding release code to v1.

Once this is merged we can push a new tag `v1.2.2` and the "releaser" will publish the required artifacts which TF registry should be able to pick.

**Comment by yussufsh** [comment]
Version `v1.2.2` is available at https://registry.terraform.io/providers/community-terraform-providers/ignition now.
